presto-main/src/main/java/com/facebook/presto/operator/scalar/sql/ArraySqlFunctions.java

```diff
@@ -37,6 +37,22 @@ public static String arrayAverage()
                 "s -> if(s[2] = 0, cast(null as double), s[1] / cast(s[2] as double)))";
     }
 
+    @SqlInvokedScalarFunction(value = "array_split", deterministic = true, calledOnNullInput = false)
+    @Description("Returns an array of arrays splitting input array into slices of given length. Ignores null elements.")
```

Comment:
what happens if the array size is not evenly divisible by the slice size? This should be documented.